data/remnant/remnant missions.txt

```diff
@@ -940,6 +940,143 @@ mission "Remnant: Learn Sign Follow Up"
+			`	"Thank you, Captain. If you ever come across any other interesting information like this, don't hesitate to bring it to me."`
+
+
+
+# The B mission is for players that kept the Remnant secret from the Deep, but did the wormhole research.
+mission "Remnant: Follow-up on Terminus Researchers B"
+	name "Check up on Wormhole Researchers"
+	description "Return to Bounty to check up on a group of wormhole researchers you met some time ago."
+	minor
+	source
+		government Remnant
+	destination "Valhalla"
+	to offer
+		has "Remnant: Learning Sign 2: done"
+		has "Deep: Wormhole Research: done"
+		not "Remnant: Follow-up on Terminus Researchers A: done"
+		not "Remnant: Follow-up on Terminus Researchers B: active"
+		random < 40
+	on offer
+		conversation
+			`Looking around at the Remnant spaceport, it suddenly strikes you how amazing this is. You never would have imagined that you would end up here back when you earned your pilot's license. It seems so long ago that your biggest concern was deciding whether to pay off your loan by ferrying cargo, doing courier jobs, or escorting merchants. (and on your darker days - contemplating being a pirate).`
+			`	Thinking back, you recall how you helped Ivan and a team of researchers from the Deep study the Ember Waste threshold wormhole. Research like that might be of interest to the Remnant, seeing as it could open the door to research vessels venturing into the Ember Wastes.`
+			choice
+				`	(Find a prefect.)`
+				`	(Forget about it.)`
+					decline
+			`	For lack of a better place to go, you stop by the information desk where you picked up the sign language course. You tell the attendant that you have something potentially important - but non-urgent - to discuss in regards to Remnant security. They tap a few buttons, then give you directions down a hallway. You quickly find yourself in an armory where several remnant guards are in the process of cleaning their weapons. One stands up and comes over to you.`
+			`	"Greetings Captain, I am Prefect Torza. I was told you had some information that could be important?"`
+			`	He takes a seat on a bench on the far side of the room from the other Remnant, and gestures for you to take a seat with him.`
+			`	You give him some background about the current state of Human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers studying the wormhole. He appreciates that you kept them focused on the wormhole train instead of revealing the Remnant, but still appears concerned.`
```

Comment:
```suggestion
			`	You give him some background about the current state of human and Hai understanding about the wormholes - or the lack thereof - then explain how you met a team of researchers studying the wormhole. He appreciates that you kept them focused on the wormhole train instead of revealing the Remnant, but still appears concerned.`
```